_gsdocs-proposals/2020/proposal_CVMFSPodmanIntegration.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Podman CVMFS integration
+layout: default
+project: CernVM-FS
+year: 2020
+organization:
+ - CERN
+---
+
+## Description
+
+[CernVM-FS (CVMFS)][cvmfs] is a globally-distributed filesystem used to lazily distribute software in different server farm.
+
+[Podman][podman] is an utility to run and use containers.
+It provides the same command line interface than Docker but it runs without the need of a privileged daemon.
+These two characteristics make it extremely interesting for workload used in scientific data-centers.
+
+It has been shown that only a small portion of all the files in a container images is necessary to run the image itself.
+This is even more accentuated in scientific container images since they usually include big binaries, not always used by the specific task.
+Our goal is to merge the lazy load capabilities of CVMFS with the container workflow allowed by podman, to quickly load big scientific container images while maintaining the isolation and convenience of containers.
+
+There is already an integration for Docker, and another for __containerd__ (kubernetes) is about ready.
+
+The filesystem in CVMFS has the following structure:
```

Comment:
A CernVM-FS file system hosting container images and layers shows the following structure. From a distribution point of view, in CernVM-FS a layer (or image) is a directory containing the unpacked files rather than a single tarball.